### PR TITLE
Implement Chargeback resource

### PR DESCRIPTION
**Pull request by vernondegoede**
- Support `https://api.mollie.nl/v1/chargebacks` endpoint (https://www.mollie.com/nl/docs/reference/chargebacks/list-all)
- Added unit tests
- Added integration tests

### CODE REVIEW

File: tests/unit/models/chargeback.test.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import Chargeback from 'models/chargeback';
+
+describe('customer model', () => {
```
Comment (lvgunst) [inline tests/unit/models/chargeback.test.js]: Fix model name

Comment (vernondegoede) [reply tests/unit/models/chargeback.test.js]: Oops, renamed. ✅

File: tests/integration/chargebacks.test.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+axios.defaults.adapter = httpAdapter;
+
+/**
+ * Load the API_KEY environment variable
+ */
+dotenv.config();
+
+const mollieClient = mollie({ apiKey: process.env.API_KEY });
+
+describe('customers', () => {
```
Comment (dabroek) [inline tests/integration/chargebacks.test.js]: Customers?

**Comment by vernondegoede** [comment]
I'm not sure why the Travis build is failing. It's working fine locally using a `.env` file. Travis should be adding the `API_KEY` as a secret env variable, but for some reason this doesn't work. 

I asked @wvdeutekom to look into this.

File: tests/integration/chargebacks.test.js (hunk 2 above)
Comment (vernondegoede) [reply tests/integration/chargebacks.test.js]: Fixed.

**Comment by dabroek** [comment]
I don't think this should be a top level resource. Chargeback is a sub-resource. You should implement these:
```
v1/payments/<payment_id>/chargebacks
v1/payments/<payment_id>/chargebacks/<chargeback_id>
```

**Comment by vernondegoede** [comment]
@dabroek I'll add the chargeback subresources in another PR, as discussed.

File: tests/integration/chargebacks.test.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ */
+axios.defaults.adapter = httpAdapter;
+
+/**
+ * Load the API_KEY environment variable
+ */
+dotenv.config();
+
+const mollieClient = mollie({ apiKey: process.env.API_KEY });
+
```
Comment (wvdeutekom) [inline tests/integration/chargebacks.test.js]: @vernondegoede in order to start debugging this you will have to print two things here:
* All environment variables this application can read, this way we know if `API_KEY` is set at all in the pipeline (even though the settings say it is)
* The full mollieClient object, so we can see if the apiKey variable is set if the env variable does exist

Comment (lvgunst) [reply tests/integration/chargebacks.test.js]: I don't think we ever set an `API_KEY` environment variable in Travis config (using a encrypted variable): https://docs.travis-ci.com/user/environment-variables/#Defining-encrypted-variables-in-.travis.yml

@dabroek Have you ever get this to work on the `mollie:next` branch?

Comment (dabroek) [reply tests/integration/chargebacks.test.js]: Yes I had it working before. I put the environment variable there myself.

Comment (wvdeutekom) [reply tests/integration/chargebacks.test.js]: I could see the `API_KEY` in the settings, except for the value assigned to it.

**Comment by RickWong** [comment]
LGTM! 🎉

**Pull request closed without merging**